### ISSUE TITLE
Is there a missing element below Regarima residuals in Diagnostics?

### ISSUE THREAD

**Issue by TanguyBarthelemy** [jdemetra/jdplus-main#490]
*Issue revealed by Nikolina Rizanovska*

There is a white line under Regarima residuals (in Diagnostics section):

![image](https://github.com/user-attachments/assets/95986da4-46fe-4940-917b-e74cbf50f5ed)

Is there a missing element that cannot be displayed for any reason?
Probably `spectral td peaks` should be there?

Thank you!

**Comment by TanguyBarthelemy** [comment]
Related to quaterly series.

**Comment by palatej** [comment]
No. That's just because those lines are related to different groups of diagnostics (which can be enabled/disabled separately).

**Comment by TanguyBarthelemy** [comment]
So here, there is no test for `spectral td peaks`?
Is it related to the frequency? Or the data?
